### PR TITLE
Feature/new navbar notifications

### PR DESCRIPTION
**Pull request by JDownloader**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -82,14 +81,14 @@
...
-    "@bower_components/ovh-api-services": "ovh-ux/ovh-api-services#^3.0.0",
+    "@bower_components/ovh-api-services": "https://github.com/ovh-ux/ovh-api-services.git#feature/new-navbar-staging",
     "@bower_components/ovh-common-style": "ovh-ux/ovh-common-style#^3.2.2",
     "@bower_components/ovh-jquery-ui-draggable-ng": "ovh-ux/ovh-jquery-ui-draggable-ng#^0.0.5",
     "@bower_components/ovh-manager-webfont": "ovh-ux/ovh-manager-webfont#^1.0.1",
     "@bower_components/uri.js": "medialize/URI.js#~1.15.0",
     "@bower_components/validator-js": "chriso/validator.js#~3.37.0",
-    "ovh-ui-angular": "ovh-ux/ovh-ui-angular#^2.2.0",
-    "ovh-ui-kit": "ovh-ux/ovh-ui-kit#^2.2.0",
+    "ovh-ui-angular": "https://github.com/ovh-ux/ovh-ui-angular.git#feature/new-navbar-staging",
```
Comment (JDownloader) [inline package.json]: I'm waiting for a new ui-kit release

File: client/components/navbar/navbar-notification.service.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        notification.acknowledged = _.contains(["acknowledged", "completed", "unknown"], notification.status);
+        notification.actionClicked = toUpdate => this._toggleSublinkAction(toUpdate);
+        notification.linkClicked = toUpdate => this._toggleSublinkAction(toUpdate, true);
+        return notification;
+    }
+
+    aknowledgeAll () {
+        if (this.navbarContent) {
+            const toAcknowledge = [];
+            this.navbarContent.subLinks.forEach(subLink => {
```
Comment (FredericEspiau) [inline client/components/navbar/navbar-notification.service.js]:
```javascript
const subLinksToAcknoledge = this.navbarContent.subLink
	.filter((subLink) => !subLink.acknowledged && subLink.isActive);

return this.OvhApiNotificationAapi
	.post({ acknowledged: subLinksToAcknoledge.map((subLink) => subLink.id) }).$promise
	.then(() => {
    	subLinksToAcknoledge.forEach(sublink => {
        	sublink.acknowledged = true;
        });
    });
```

File: client/components/navbar/navbar-notification.service.js [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+                this.OvhApiNotificationAapi.post({ acknowledged: toAcknowledge.map(x => x.id) }).$promise.then(() => {
+                    toAcknowledge.forEach(sublink => {
+                        sublink.acknowledged = true;
+                    });
+                });
+            }
+        }
+    }
+
+    _setRefresfTime (sublinks) {
```
Comment (FredericEspiau) [inline client/components/navbar/navbar-notification.service.js]: `Refresf` 😃

File: client/app/common/translations/Messages_fr_FR.xml [hunk 4]
```diff
@@ -87,7 +87,10 @@
    <translation id="common_menu_support_email_history" qtlid="260091">Historique des emails</translation>
    <translation id="common_menu_support_telephony_contact" qtlid="260827">Assistance téléphonique</translation>
 
-
+   <translation id="common_navbar_notification_title" qtlid="33250">Notifications</translation>
+   <translation id="common_navbar_notification_error" qtlid="323434">Oups, nous n'arrivons pas à initialiser le menu !</translation>
```
[the remark this answers is not in the document]
Comment (JDownloader) [reply client/app/common/translations/Messages_fr_FR.xml]: ovh-angular-module-status, which will be depreciated

File: client/app/app.js [hunk 5]
```diff
@@ -178,6 +175,24 @@ angular.module("managerApp", [
...
+                notification: {
+                    markRead: $translate.instant("common_navbar_notification_mark_as_read"),
+                    markUnread: $translate.instant("common_navbar_notification_mark_as_unread")
+                }
+            };
+
+            ouiPaginationConfiguration.translations = {
+                resultsPerPage: $translate.instant("common_pagination_resultsperpage"),
+                ofNResults: $translate.instant("common_pagination_ofnresults")
+                    .replace("TOTAL_ITEMS", "{{totalItems}}"),
```
[the remark this answers is not in the document]
Comment (JDownloader) [reply client/app/app.js]: I didn't wrote that, I just move the code to follow an alphabetical order

File: client/components/navbar/navbar-notification.service.js (hunk 3 above)
Comment (JDownloader) [reply client/components/navbar/navbar-notification.service.js]: lol, will correct that

File: package.json (hunk 1 above)
Comment (FredericEspiau) [reply package.json]: It's been released ! 🎉

**Comment by FredericEspiau** [comment]
@JDownloader conflits 😄

**Comment by JDownloader** [comment]
Will be done in a new PR https://github.com/ovh-ux/ovh-manager-cloud/pull/751

**Pull request closed without merging**